[2023-08-31 13:21:46,274] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-08-26T00:00:00+00:00 [queued]>
[2023-08-31 13:21:46,323] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-08-26T00:00:00+00:00 [queued]>
[2023-08-31 13:21:46,324] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:21:46,325] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:21:46,325] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:21:46,386] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-08-26T00:00:00+00:00
[2023-08-31 13:21:46,390] {standard_task_runner.py:52} INFO - Started process 2082 to run task
[2023-08-31 13:21:46,409] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-08-26T00:00:00+00:00', '--job-id', '88', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpe8d7fzx0', '--error-file', '/tmp/tmpep2tj7z8']
[2023-08-31 13:21:46,422] {standard_task_runner.py:77} INFO - Job 88: Subtask second_task
[2023-08-31 13:21:46,557] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-08-26T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:21:46,695] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-26T00:00:00+00:00
[2023-08-31 13:21:46,698] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:21:46,703] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:21:46,748] {bash.py:169} INFO - Output:
[2023-08-31 13:21:46,791] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:21:46,795] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:21:46,874] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210826T000000, start_date=20230831T132146, end_date=20230831T132146
[2023-08-31 13:21:47,257] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:21:47,296] {local_task_job.py:146} INFO - Task exited with return code 0
